controllers/competition_supervisor/competition_supervisor.py

```diff
@@ -24,6 +24,8 @@ def recording_path(when: Optional[datetime.datetime] = None) -> Path:
 
     date = when.date().isoformat()
     name = '{}.html'.format(when.isoformat())
+    # Windows doesn't like colons in filenames
+    name = name.replace(':', '-')
```

Comment:
Hyphens already have a meaning in ISO 8601 formatted datetimes, specifically that they're the `YYYY-MM-DD` part separators. Maybe consider using something else (perhaps dots)?